.gitignore

```diff
@@ -10,3 +10,4 @@ test/dummy/.sass-cache
 coverage/
 *.gem
 Gemfile.lock
+.idea
```

Comment:
You should add this to your [global .gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore).